odoo/fields.py

```diff
@@ -1747,10 +1779,8 @@ def to_datetime(value):
                     raise ValueError("Datetime field expects a naive datetime: %s" % value)
                 return value
             return datetime.combine(value, time.min)
-        try:
-            return datetime.strptime(value[:DATETIME_LENGTH], DATETIME_FORMAT)
-        except ValueError:
-            return datetime.strptime(value, DATE_FORMAT)
+
+        return datetime.strptime(value, DATETIME_FORMAT[:len(value)-2])
```

Comment:
Hey there, I'm running in an issue because of this `DATETIME_FORMAT[:len(value)-2]`.

I defined a cron job to run everyday at 8 using `<field name="nextcall" eval="(datetime.now() + timedelta(days=1)).strftime('%Y-%m-%d 8:00:00')" />`

But when it tries to convert the string to a datetime object, I get the following error : `ValueError: stray % in format '%Y-%m-%d %H:%M:%'`